website/thaliedje/migrations/0010_marietjeplayer_remove_player_client_id_and_more.py

```diff
@@ -0,0 +1,117 @@
+# Generated by Django 4.1.4 on 2023-01-17 13:27
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("thaliedje", "0009_playerlogentry_thaliedjecontrolevent"),
```

Comment:
This migration does not migrate old Spotify clients to new ones.

Reply:
Can't be fixed in a nice manner